api.planx.uk/saveAndReturn/resumeApplication.ts

```diff
@@ -81,16 +85,19 @@ const validateRequest = async (
         }
       }
     `;
-    const { lowcal_sessions, teams } = await adminClient.request(query, {
-      teamSlug,
-      email: email.toLowerCase(),
-    });
+    const { sessions, teams } = await $api.client.request<ValidateRequest>(
+      query,
+      {
+        teamSlug,
+        email: email.toLowerCase(),
+      },
+    );
 
     if (!teams?.length) throw Error;
 
     return {
       team: teams[0],
-      sessions: lowcal_sessions,
+      sessions: sessions || [],
```

Comment:
same comment as above here - this re-naming feels like it changes the whole table reference!